File: wk6Ex2b.py
def get_value_from_array(array, index):
    # Ensure the index is within the bounds of the array
    if index < 0 or index >= len(array):
        raise IndexError("Index out of bounds")
    # Return the element from the array at the specified index
    return array[index]

def main():
    items = [902, 903, 904, 905, 906]
    for i in range 16):
        try:
            ret_val = get_value_from_array(items, i)
            print(f"Returned value at index {i} is {ret_val}")
        except IndexError:
            print(f"Index {i} is out of bounds")

if __name__ == "__main__":
    main()

File: wk6Ex2b.c
#include <stdio.h>
#include <stdlib.h>

int getValueFromArray(int *array, int len, int index)
{
    // Return the element from the array at the specified index
    return array[index];
}

int main(int argc, char **argv)
{
    int items[] = {902, 903, 904, 905, 906};
    int retVal = 0;

    for (int i = 0; i <= 15; i++)
    {
        retVal = getValueFromArray(items, sizeof(items), i);
        printf("Returned value at index %d is %d\n", i, retVal);
    }
}
